front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -19,22 +18,79 @@
     }
   }
 
+  .micro-macro-buttons {
+    button {
+      color: var(--grey60);
+    }
+    font-size: 0.875rem;
+    letter-spacing: 0.1019rem;
+    line-height: 1.125rem;
+    margin-block: 1.813rem 2.063rem;
+    display: inline-flex;
+    background-color: var(--ambiantB10);
+    border-radius: 0.375rem;
+    padding: 0.25rem;
+
+    box-shadow:
+      0 0.125rem 0.125rem -0.0625rem rgba(255, 171, 88, 0.27),
+      0 0.0625rem 0.125rem rgba(0, 0, 0, 0.16);
+    height: 2rem;
+    .micro-macro-separator {
+      width: 0.0625rem;
+      margin-inline: 0.625rem;
+      background-color: var(--grey20);
+    }
+    .micro-button {
+      margin-top: 0.1875rem;
+      text-align: center;
+      padding-left: 1rem;
+      height: 1.5rem;
```

Comment:
height should be 18px. Also it seems lile the text is a little to low comparing to the mockup, maybe we should try to remove 1px from the margin top